integration/network/network_test.go

```diff
@@ -167,3 +172,68 @@ func TestNetworkInspectWithScope(t *testing.T) {
 	_, err = cli.NetworkInspect(ctx, name, client.NetworkInspectOptions{Scope: "local"})
 	assert.Check(t, is.ErrorType(err, cerrdefs.IsNotFound))
 }
+
+func TestBridgeAndCustomNetworkAttach(t *testing.T) {
+	skip.If(t, testEnv.RuntimeIsWindowsContainerd(),
+		"Skipping test: fails on Containerd due to unsupported platform request error during NetworkConnect operations")
+
+	ctx := setupTest(t)
+	apiClient := testEnv.APIClient()
+
+	const name = "testnetwork"
+	const containerName = "test-container"
+
+	network.CreateNoError(ctx, t, apiClient, name)
+	defer network.RemoveNoError(ctx, t, apiClient, name)
+
+	// verifying the new network doesn't attach any container
+	nr, err := apiClient.NetworkInspect(ctx, name, client.NetworkInspectOptions{})
+	assert.NilError(t, err)
+	assert.Equal(t, nr.Network.Name, name)
+	assert.Equal(t, len(nr.Network.Containers), 0)
```

Comment:
It's worth thinking about whether each assertion needs to stop the test (here, in the rest of this test, and in-general).

In this case, if the inspect fails, things are probably so broken it's not worth continuing.

But, in the unlikely event there's a container attached to the new network - might as well let the test carry on to see what else happens.

```suggestion
	assert.NilError(t, err)
	assert.Check(t, is.Equal(nr.Network.Name, name))
	assert.Check(t, is.Equal(len(nr.Network.Containers), 0))
```